b2sdk/account_info/sqlite_account_info.py

```diff
@@ -39,7 +42,7 @@ class SqliteAccountInfo(UrlPoolAccountInfo):
     completed.
     """
 
-    def __init__(self, file_name=None, last_upgrade_to_run=None):
+    def __init__(self, file_name=None, last_upgrade_to_run=None, profile=None):
```

Comment:
```suggestion
    def __init__(self, file_name=None, last_upgrade_to_run=None, profile: Optional[str] = None):
```

Reply:
I just follow the convention to not use type hints in b2sdk code :)